invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordVersionsList.js

```diff
@@ -28,22 +28,20 @@ const RecordVersionItem = ({ item, activeVersion }) => {
     <>
       <Grid.Row
         key={item.id}
-        columns={2}
+        columns={1}
         {...(activeVersion && { className: "version-active" })}
       >
-        <Grid.Column width={10}>
+        <Grid.Column>
+        <small className="text-muted" style={{ float: "right" }}>
+             {item.publication_date}
+           </small>
           <a href={`/records/${item.id}`}>Version {item.version}</a>
-          {doi && (
-            <small className="text-muted" style={{ float: "left" }}>
-              {doi}
+          {<br/>}{doi && (
+            <small  className="text-muted">
```

Comment:
extra space 
```suggestion
            <small className="text-muted">
```

Reply:
I did another commit, and now if the window is very small it will break in both cases @ppanero 
![break](https://user-images.githubusercontent.com/30771373/117032305-ab925a80-ad01-11eb-8a12-b60b0a6b59a3.png)
![nobreak](https://user-images.githubusercontent.com/30771373/117032318-ae8d4b00-ad01-11eb-9ec3-ed2141d9d004.png)